### ISSUE TITLE
readme.md lead me down a rabbit hole

### ISSUE THREAD

**Issue by emteknetnz** [silverstripe/silverstripe-textextraction#43]
Documentation in readme.md says that we should update yaml files to get things working

However this lead me down a rabbit hole of not working when deploying to CWP

The best way to get this working locally and on CWP, is to simply add the following to your _ss_environment.php

define('SS_TIKA_ENDPOINT', 'http://localhost:9998');

and it worked with fulltextsearch-localsolr + tika

doing the yml things the readme said simply messed things up

**Comment by robbieaverill** [comment]
Hey @emteknetnz - this module is only half updated, so it won't work yet

**Comment by robbieaverill** [comment]
Fair enough, particularly because this module is mostly for CWP. Do you feel like updating the docs?

**Comment by GuySartorelli** [comment]
Is the documentation about the yaml config outright wrong (and should be removed or replaced) - or would it be safe to keep that as is, and just add the env variable as the recommended configuration? (I realise this was opened for a much older version but the docs have barely changed since then and the environment variable is still the main way to configure this module - at least in my experience)
